clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesNamedAccountCredentialsInitializerSpec.groovy

```diff
@@ -1,149 +0,0 @@
-/*
- * Copyright 2019 Google, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License")
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.netflix.spinnaker.clouddriver.kubernetes.v2.security
-
-import com.netflix.spectator.api.NoopRegistry
-import com.netflix.spinnaker.cats.module.CatsModule
-import com.netflix.spinnaker.clouddriver.data.ConfigFileService
-import com.netflix.spinnaker.clouddriver.kubernetes.config.KubernetesConfigurationProperties
-import com.netflix.spinnaker.clouddriver.kubernetes.config.LinkedDockerRegistryConfiguration
-import com.netflix.spinnaker.clouddriver.kubernetes.security.KubernetesNamedAccountCredentials
-import com.netflix.spinnaker.clouddriver.kubernetes.security.KubernetesNamedAccountCredentialsInitializer
-import com.netflix.spinnaker.clouddriver.kubernetes.v1.security.KubernetesV1Credentials
-import com.netflix.spinnaker.clouddriver.kubernetes.v2.names.KubernetesManifestNamer
-import com.netflix.spinnaker.clouddriver.kubernetes.v2.op.job.KubectlJobExecutor
-import com.netflix.spinnaker.clouddriver.names.NamerRegistry
-import com.netflix.spinnaker.clouddriver.security.AccountCredentialsRepository
-import com.netflix.spinnaker.clouddriver.security.ProviderVersion
-import spock.lang.Specification
-
-class KubernetesNamedAccountCredentialsInitializerSpec extends Specification {
```

Comment:
These tests were all recently added as part of a refactor that I did on the account creation code.  While the specific class they're testing might be gone, the business logic of the tests is still valid so these tests should be moved/modified rather than just deleted.